src/app/config/app.config.ts

```diff
@@ -14,7 +14,24 @@ import {ErrorHelper} from '../utilities/error-helper';
 @Injectable()
 export class AppConfig {
 
+  public static DEFAULT_API_URL = 'https://transmart-dev.thehyve.net';
```

Comment:
Should this have a default? I would see this as always environment specific.

Reply:
yes, we should always specify this in config.default.json, on an off-chance that this url is not there, would be nice to have sth. to fall back on?